src/console.cpp

```diff
@@ -221,7 +221,7 @@ static std::string &RemoveUnderscores(std::string &name)
  */
 /* static */ void IConsole::CmdRegister(std::string name, IConsoleCmdProc *proc, IConsoleHook *hook)
 {
-	IConsole::Commands().try_emplace(RemoveUnderscores(std::string(name)), name, proc, hook);
```

Comment:
This one is intended so the key is a duplicate of the name.
That way key doesn't contain '_' but displayed list still has them (and matches the `help` command output).

Reply:
You cannot use `RemoveUnderscores()` on an `rvalue`, per its signature.

If you need a copy, either:
- Overload `RemoveUnderscores()` to include an object copy definition
- Insert the object 1st and act on it through `RemoveUnderscores`